packages/patternfly-4/react-core/src/components/Popover/Popover.tsx

```diff
@@ -69,6 +78,8 @@ export interface PopoverProps {
    * Instead, the consumer is responsible for closing the popover themselves by adding a callback listener for the shouldClose prop.
    */
   isVisible?: boolean;
+  /** The events which cause the popover to show. **/
+  triggers?: Triggers[];
```

Comment:
Can you please note that is isVisible is set, this prop is ignored.

Reply:
Good idea!